client/app/hosting/runtimes/add/hosting-runtimes-add.controller.js

```diff
@@ -45,9 +47,8 @@ angular.module('App').controller(
 
     isValid() {
       if (
-        this.entryToCreate
-        && this.entryToCreate.type
-        && this.entryToCreate.type.indexOf('nodejs') !== -1
+        _.has('this.entryToCreate.type')
```

Comment:
Your test won't return true, use it like this :
```suggestion
        _.has(this, 'entryToCreate.type')
```